src/huggingface_hub/utils/_git_credential.py

```diff
@@ -37,11 +38,12 @@ def list_credential_helpers(folder: Optional[str] = None) -> List[str]:
         # NOTE: If user has set an helper for a custom URL, it will not we caught here.
         #       Example: `credential.https://huggingface.co.helper=store`
         #       See: https://github.com/huggingface/huggingface_hub/pull/1138#discussion_r1013324508
-        return sorted(  # Sort for nice printing
-            {  # Might have some duplicates
-                line.split("=")[-1].split()[0] for line in output.split("\n") if "credential.helper" in line
-            }
-        )
+        for line in output.split("\n"):
+            l_cred_helper = sorted(  # Sort for nice printing
+                # Might have some duplicates
+                re.findall(r"^\s*credential\.helper\s*=\s*(\w*)\s*$", line)
+            )
+        return l_cred_helper
```

Comment:
Using `re.findall` is good. However, you don't need to run it on each line one by one. You can run on the whole output at once and use `re.MULTILINE` as flag. Here is how it looks like when you apply it only once:

```suggestion
        return sorted(  # Sort for nice printing
        set(  # Might have some duplicates
            re.findall(
                r"^\s*credential\.helper\s*=\s*(\w+)\s*$",
                output,
                flags=re.MULTILINE | re.IGNORECASE,
            )
        )
    )
```

Reply:
Thanks @Wauplin for the feedback. I've incorporated the above changes as indicated.